### DISCUSSION TITLE
Add support for mouse button 4&5

### DISCUSSION THREAD

**Discussion by jihuayu** [spacedriveapp/spacedrive#1880] [Feature Request]
Spacedrive is a good project, and I really like it. But, I have noticed some uncomfortable aspects while using it.
I always use mouse button 4&5 to navigate back or forward, but it doesn't seem to exist in Spacedrive.
Can you add this feature in it?

Thank you for creating such a great project.

![image](https://github.com/spacedriveapp/spacedrive/assets/32858525/d231fd27-ebc2-4f73-ac9c-63b607f1e1a4)

**Comment by TheIceMagmaCube** [comment]
I agree.
I use these buttons a lot. This could a good accessibilty feature !

**Comment by utkubakir** [comment]
We actually do have this but it's not working for all mouses for some reason. We will look into it again in the future... #1957

**Comment by ameer2468** [comment]
This has been resolved - until Tauri makes a new release, Linux may not be supported in the meantime. 

Please give it a go and test in the next upcoming release.

Thank you :)

**Reply by jihuayu** [reply]
Wow, It's very good. Thank you! I love it!
